sphinx/application.py

```diff
@@ -975,6 +990,16 @@ def add_css_file(self, filename: str, **kwargs: str) -> None:
             # => <link rel="alternate stylesheet" href="_static/fancy.css"
             #          type="text/css" title="fancy" />
 
+        .. list-table:: priority range for CSS files
+           :widths: 20,80
+
+           * - Priority
+             - Main purpose in Sphinx
```

Comment:
Note: At present, Sphinx's CSS files are embeded into theme files directly. So it is hard to control its order. I'll improve it in another PR.

Reply:
Note: I posted https://github.com/sphinx-doc/sphinx/pull/8643.